### ISSUE TITLE
Wrong paths in package.json

### ISSUE THREAD

**Issue by Bessonov** [cvara/rxdb-hooks#32]
Hi Christoforos :handshake: 

I'm not sure, but it seems like something went wrong with the build. `package.json` contains following entries:
```json
	"version": "3.0.0",
[...]
	"main": "./dist/index.js",
	"types": "./dist/index.d.ts",
```
but files are under `dist/src`:
```
$ find -L packages/frontend/node_modules/rxdb-hooks -name "index.*"
packages/frontend/node_modules/rxdb-hooks/dist/src/index.js
packages/frontend/node_modules/rxdb-hooks/dist/src/index.d.ts
packages/frontend/node_modules/rxdb-hooks/src/index.ts
```

Notice also the last entry. I think it shouldn't be here.

**Comment by Bessonov** [comment]
Excellent, thank you!

**Comment by cvara** [comment]
Hi @Bessonov.

Thanks for reporting this. Just published version `3.0.1` that fixes the issue (a minor tsconfig misconfiguration that I overlooked).
